bescort.lic

```diff
@@ -966,28 +966,47 @@ class Bescort
 
     path = mode == 'shard' ? $ICE_PATH_SHARD : $ICE_PATH_HIB
 
-    if exists?('skates')
-      EquipmentManager.new.empty_hands
-      footwear = get_settings.footwear
-      fput("remove #{footwear}") if footwear
-      bput("stow my #{footwear}", 'You put') if footwear
-      fput('get skates')
-      fput('wear skates')
-      path.each do |movement|
-        move(movement)
+    EquipmentManager.new.empty_hands
+    footwear = get_settings.footwear
+    move_fast = false
+
+    if DRCI.exists?("skates")
+      if !footwear
+        DRCI.get_item?("skates") && DRCI.wear_item?("skates")
+            move_fast = true
+      elsif footwear
+        if DRCI.remove_item?(footwear)
+          if DRCI.put_away_item?(footwear)
+            if DRCI.get_item?("skates") && DRCI.wear_item?("skates")
+              move_fast = true
+            else
+              DRCI.put_away_item?("skates")
+            end
+          else
+            DRCI.wear_item?(footwear)
+          end
+        else
+          DRCI.put_away_item?("skates")
```

Comment:
I don't think this `else` block to put skates away is necessary because you wouldn't have pulled out the skates at that point.